library/src/main/java/com/github/twocoffeesoneteam/glidetovectoryou/SvgSoftwareLayerSetter.java

```diff
@@ -41,11 +41,9 @@ public boolean onLoadFailed(GlideException e, Object model, Target<PictureDrawab
     @Override
     public boolean onResourceReady(PictureDrawable resource, Object model,
                                    Target<PictureDrawable> target, DataSource dataSource, boolean isFirstResource) {
-        ImageView view = ((ImageViewTarget<?>) target).getView();
```

Comment:
You might want to keep this as is for backward compatibility as well

Reply:
Yes. I didn't consider backward compatibility. Have modified the same.